tox.ini

```diff
@@ -25,6 +26,19 @@ basepython=
     mypy: mypy
```

Comment:
This line is super weird, and should probably be removed. (setting the base python to mypy, but only in a mypy env which doesn't exist)

Reply:
Ah leftover from hair pulling attempts to figure out what was going on with this segfault.